src/mono/CMakeLists.txt

```diff
@@ -747,10 +747,9 @@ elseif(HOST_BROWSER)
   set(STATIC_ICU 1)
   set(ICU_LIBS "icucore")
 elseif(HOST_IOS OR HOST_TVOS OR HOST_MACCAT)
-  set(ICU_FLAGS "-DTARGET_UNIX -DU_DISABLE_RENAMING -Wno-reserved-id-macro -Wno-documentation -Wno-documentation-unknown-command -Wno-switch-enum -Wno-covered-switch-default -Wno-extra-semi-stmt -Wno-unknown-warning-option -Wno-deprecated-declarations")
+  set(ICU_FLAGS "-DTARGET_UNIX -DTARGET_IOS -DTARGET_TVOS -DTARGET_MACCATALYST -DU_DISABLE_RENAMING -Wno-reserved-id-macro -Wno-documentation -Wno-documentation-unknown-command -Wno-switch-enum -Wno-covered-switch-default -Wno-extra-semi-stmt -Wno-unknown-warning-option -Wno-deprecated-declarations")
```

Comment:
is it ok if we pass `-DTARGET_IOS -DTARGET_TVOS -DTARGET_MACCATALYST` unconditionally? I'd have expected we only pass on the respective platform

Reply:
Updated, thanks.